pkg/mount/mount.go

```diff
@@ -120,13 +138,150 @@ func (m *mounter) getDevicePathBySerialID(volumeID string) (string, error) {
 			return source, nil
 		}
 		if !os.IsNotExist(err) {
+			fmt.Printf("Not found: %s\n", err.Error())
```

Comment:
AFAIK, The error here can also be due to other reasons like permission denied, IO errors, etc.

Reply:
the error being printed will provide details as to why it failed